style.css

```diff
@@ -0,0 +1,503 @@
+}
+
+.more-articles-grid {
+  width: 880px;
+  display: grid;
+  grid-template-columns: 280px 280px 280px;
+  grid-template-rows: 1fr 1fr;
+  grid-gap: 15px;
+  border-bottom: 0.5px solid black;
+  padding-bottom: 30px;
+  padding-top: 30px;
+  margin-bottom: 30px;
+}
+
+.more-article-single {
+  width: 280px;
+}
+
+.more-article-single figure img {
+  width: 100%;
+  height: 150px;
+}
+
+.more-article-single figure figcaption {
+  text-align: right;
+  width: 280px;
+  font-size: 10px;
+  padding-bottom: 10px;
+}
+
+.more-articles-grid-bottom {
+  width: 880px;
+  display: grid;
+  grid-template-columns: 280px 280px 280px;
+  grid-template-rows: 1fr;
+  grid-gap: 15px;
+  padding-bottom: 30px;
+  padding-top: 30px;
+  margin-bottom: 30px;
+}
+
+.more-article-single h6 {
+  font-family: monospace;
+  color: #8b888b;
+  margin-top: 5px;
+}
+
+/* MOST POPULAR SIDE BAR */
+
+.more-popular-div p {
+  border-top: 0.5px solid rgb(226, 226, 226);
+  padding: 15px 0;
+}
+
+.most-popular-side-bar {
+  border-top: 0.5px solid black;
+  width: 400px;
+  padding: 25px 0 0 0;
+  margin-left: 25px;
+}
+
+.most-popular-side-bar h4 {
+  padding-bottom: 20px;
+}
+
+/* FOOTER */
+
+footer {
+  width: 70%;
+  border-top: 4px double gray;
+  margin: 50px auto;
+}
+
+.footer-head {
+  justify-content: space-between;
+  align-items: center;
+}
+
+.footer-head-image {
+  width: 200px;
+  height: auto;
+  margin: 10px 0;
+}
+
+.footer-head a {
+  padding-top: 10px;
+  text-decoration: none;
+  color: #5ba4da;
+}
+
+.footer-list {
+  width: 100%;
+  justify-content: space-between;
+}
+
+.subscribe-div {
+  border-left: 1px solid gray;
+  padding-left: 15px;
+}
+
+.subscribe-line {
+  border-bottom: 1px solid gray;
+  padding-bottom: 5px;
+}
+
+.footer-list h4 {
+  margin: 10px 0;
+  font-weight: bold;
+}
+
+.footer-list a {
+  padding-top: 10px;
+  text-decoration: none;
+  color: black;
+}
+
+.subscribe-div div i {
+  padding-right: 5px;
+}
+
+.footer-bottom {
+  justify-content: space-between;
+  padding: 20px 70px;
+  border-top: 1px solid grey;
+  margin: 20px 0;
+}
+
+.footer-bottom a {
+  font-size: 10px;
+  text-decoration: none;
+  color: grey;
+}
+
+.subscribe-div div a {
+  font-size: 14px;
+  font-weight: bold;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+
+/* HOVER EFFECTS */
+
+.external-links:hover {
+  text-decoration: none;
+}
+
+nav ul:hover,
+.logo:hover,
+.blog-post-foot button:hover,
+.header-buttons button:hover,
+.more-article-single:hover {
+  cursor: pointer;
+}
+
+.blog-post-head i:hover,
+.blog-post-foot-icons i:hover {
+  color: #8b888b;
+  cursor: pointer;
+}
+
+.third-item-nav:hover,
+.footer-bottom a:hover {
+  text-decoration: underline;
+}
+
+article h4:hover {
+  text-decoration: underline;
+  cursor: pointer;
+}
+
+.blog-post-bottom span:hover {
+  text-decoration: none;
+  cursor: pointer;
+}
+
+.more-article-single h4:hover {
+  color: #8b888b;
+}
+
+.more-popular-div p:hover {
+  color: #8b888b;
+  cursor: pointer;
+}
```

Comment:
Some ideas:
```
.pointer {
    cursor: pointer;
}

.no-underline {
    text-decoration: none;
}

.underline {
    text-decoration: underline;
}

.color-primary {
   color: #8b888b;
}

/* --- And their corresponding :hover versions --- */
```

### [COMMENTS] :green_circle: 

The project is using ```cursor: pointer``` property to mimic and hardcode the original website appearance. While it works "visually" is better to use the link attribute or ```a``` tag also more HTML semantic.